styles/styles.css

```diff
@@ -1,3 +1,7 @@
+.navbar-brand {
+  font-size: 2rem;
```

Comment:
You can control font-size in HTML as well

Reply:
Using h1 and font-weight-light, gets the font size correct, but it crams the lines together more than my current version. Apparently I can change the line-height for Bootstrap, but do I need to download the source files? Right now I use BootstrapsCDN, so I don't have to download any of the Bootstrap files. 

The reason I ask is because the typography section says, "Use the $font-family-base, $font-size-base, and $line-height-base attributes as our typographic base applied to the <body>. These styles can be found within _reboot.scss, and the global variables are defined in _variables.scss. Make sure to set $font-size-base in rem."

https://getbootstrap.com/docs/4.1/content/typography/

However, I don't have any of the .scss files in my system right now. Not a big deal, I might still use Bootstraps global settings the way they are, but I thought I would ask, so I would know in the future.

Reply:
Have you tried `text-lg`?

Reply:
Yes, that class is used for text alignment when the screen size is lg or greater. 

https://getbootstrap.com/docs/4.1/migration/#utilities

For example if I use, text-lg-left then the text will be left aligned for large and extra large screen sizes. The font size stays the same. 

I think I need to use CSS to resize the text.

Reply:
@hdavidzhu do you get notified even if I do not tag you in a comment thread?

Reply:
Well it's hard to tell since you tagged me 😄

Reply:
I suppose you're right. The way Github does notifications, you would only get one notification for everything in this thread, even the previous comment where you were not tagged. So I'm trying again. Do you get this untagged message?

Reply:
Since I'm already subscribed, I think  I would get all notifications.

Reply:
Great! That's all I needed to know.